bin/detect

```diff
@@ -15,8 +15,8 @@
 BUILD_DIR=$1
 
 # Exit early if app is clearly not Python.
-if [ ! -f $BUILD_DIR/requirements.txt ] && [ ! -f $BUILD_DIR/conda-requirements.txt ] && [ ! -f $BUILD_DIR/conda-runtime.txt ]; then
+if [ -f $BUILD_DIR/conda-requirements.txt ] && [ -f $BUILD_DIR/conda-runtime.txt ]; then
+  echo Python/Conda
+else
```

Comment:
if I understand correctly it wasn't the desired behavior before? where python app was accepted.

Reply:
Exactly, before this change, if it detects any of those files in the app's codebase, it doesn't `exit` therefore, it continues considering the app as a conda app.

This is because in https://github.com/plotly/herokuish/blob/master/herokuish.json#L4 and https://github.com/plotly/herokuish/blob/master/herokuish.json#L9 both Python and Conda detections have the same prefix, and conda is executed first because then `conda` comes before `python` alphabetically.

And the reason why we didn't want to give conda's buildpack a prefix number higher than pythons, is that would mean we needed to use a different pip requirements file for a conda app, otherwise it'd be detected as a python app.

Reply:
thanks for the explanation!